man/manual.md

```diff
@@ -8,19 +8,20 @@ documentclass: report
 
 # Introduction
 
-Cppcheck is an analysis tool for C/C++ code. It provides unique code analysis to detect bugs and focuses on detecting undefined behaviour and dangerous coding constructs. The goal is to detect only real errors in the code (i.e. have very few false positives). Cppcheck is designed to be able to analyze your C/C++ code even if it has non-standard syntax (common in embedded projects).
+Cppcheck is an analysis tool for C/C++ code. It provides unique code analysis to detect bugs and focuses on detecting undefined behaviour and dangerous coding constructs. 
+The goal is to detect only real errors in the code, and generate as few false positives as possible. 
+Cppcheck is designed to analyze your C/C++ code even if it has non-standard syntax, as is common in for example embedded projects.
 
 Supported code and platforms:
 
-- You can check non-standard code that contains various compiler extensions, inline assembly code, etc.
-- Cppcheck should be compilable by any C++ compiler that handles the latest C++ standard.
+- Cppcheck checks non-standard code that contains various compiler extensions, inline assembly code, etc.
+- Cppcheck should be compilable by any compiler that supports the latest C++ standard.
 - Cppcheck should work on any platform that has sufficient CPU and memory.
 
-Please understand that there are limits of Cppcheck. Cppcheck is rarely wrong about reported errors. But there are
-many bugs that it doesn't detect.
+Cppcheck is rarely wrong about reported errors, but there are many bugs that it doesn't detect (yet). 
```

Comment:
I think this could be elaborated somehow. When I originally wrote that I believe I mostly meant that there are many bugs that are out of scope for different reasons..

Most logarithmic bugs will not be found by static analysis. To avoid logarithmic bugs I'd say it's more effective to test your software and design it carefully than running Cppcheck.

Cppcheck is not very deep. A dynamic analysis tool is much more effective to find deep bugs than Cppcheck.

Then there are also bugs that Cppcheck should detect that it does not detect yet. No sophisticated product is perfect. However there is work on a "bug hunting" mode that makes Cppcheck soundy.

Well.. different approaches are needed. Static analysis is not a silver bullet.

Reply:
well.. if you want we could merge your version.. and then I can try to elaborate a bit later..

Reply:
Not sure what is meant by "logarithmic bugs". Can you clarify?
Maybe best to leave out the sentence entirely, since it is clear that software is rarely complete, as you also state. Here the sentence states a trivial fact.
Either way, I object against addressing the reader directly.

Reply:
> Not sure what is meant by "logarithmic bugs". Can you clarify?

I mean common mistakes that cause bugs. The code does not really do what it should.

For instance:

    int microseconds = seconds * 10000000;

There is one zero too much in `10000000`.

Reply:
I have applied some more changes. Please review.